BREAKING.md

```diff
@@ -32,6 +32,7 @@ The `webpack-fluid-loader` utility was previously available from a package named
 - [Property removed from ContainerRuntime class](#Property-removed-from-the-ContainerRuntime-class)
 - [attach() should only be called once](#attach-should-only-be-called-once)
 - [Loader access in data stores is removed](#loader-access-in-data-stores-is-removed)
+- [Logger without tag support marked deprecated in ContainerContext](#logger-without-tag-support-marked-deprecated-in-containercontext)
```

Comment:
This happened in 0.44, not 0.45?  At any rate, you need a breaking change comment for 0.45 saying that this is actually happening (what you say below: "loggers passed to ContainerContext will need to support tagged events.").

Reply:
woops - yes this is 0.45